web/src/pages/Route/components/Step1/MetaView.tsx

```diff
@@ -205,6 +205,7 @@ const MetaView: React.FC<RouteModule.Step1PassProps> = ({ disabled, form, isEdit
             >
               <Select
                 disabled={disabled}
+                data-cy='route-redirect'
                 onChange={(parmas) => {
```

Comment:
can you also fix the typo to params in this PR?

Reply:
> can you also fix the typo to params in this PR?

Thanks for your review, and just to make sure that I should change the typo of `route-direct` right?

Reply:
`onChange={(parmas) => {`  see here it should be params i guess

Reply:
got it, fixed.

Reply:
thank you